targets/TARGET_TOSHIBA/TARGET_TMPM3HQ/device/TOOLCHAIN_GCC_ARM/tmpm3hqfdfg.ld

```diff
@@ -0,0 +1,162 @@
+/* Linker script for Toshiba TMPM3HQ */
+
+/* Linker script to configure memory regions. */
+MEMORY
+{
+  FLASH (rx) : ORIGIN = 0x00000000, LENGTH = 512K
```

Comment:
I didn't find the "MBED_APP_START" macro.. any reason not to have this defined in all the 3 toolchain's linker descriptions?

Reply:
Hi @ashok-rao, added "MBED_APP_START" macro in all 3 toolchain's linker descriptions.